packages/fe-fpm-writer/src/action/types.ts

```diff
@@ -0,0 +1,28 @@
+import { CustomElement, InternalCustomElement, Position } from '../common/types';
+
+export enum ControlType {
+    header = 'header',
+    footer = 'footer',
+    facet = '@com.sap.vocabularies.UI.v1.FieldGroup',
+    table = 'items/@com.sap.vocabularies.UI.v1.LineItem'
```

Comment:
items is a navProperty here not something specific

Reply:
@nlunets https://github.com/SAP/open-ux-tools/pull/39/commits/81370928578f68709d8457a9973d9d289c94a66e better?

Reply:
yes